Interdomain Routing FRR/BGP Transit AS/kathara-lab_bgp-transit-as-redistribute-bgp-non-deterministic_frr/lab.conf

```diff
@@ -0,0 +1,29 @@
+as10rt1[0]=A
```

Comment:
Please, add at the start of the lab.conf the metadata (you can see an example in the original lab) and feel free to put your name together with the other authors :)